src/components/Info/InfoHome.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+import { Description } from "../../reusableComponents/Description.jsx"
+
+const InfoWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr; 
+  gap: 10px; 
+  max-width: 100%;
+  padding: 5px 25px;
+  @media (min-width: 1024px) {
+    padding: 48px 100px;
+  }
+`;
+const RedLine = styled.div`
+  width: 100%; /* Line width to the entire width of the wrapper */
+  height: 16px;
+  background-color: var(--red-color);
+`;
+const InfoDescription = styled(Description)`
```

Comment:
Is this copied straight from Figma? In that case genious! This really makes styled components more easy to use than anything else!